src/main/kotlin/net/atos/zac/zaak/ZaakService.kt

```diff
@@ -66,7 +66,7 @@ class ZaakService @Inject constructor(
                         bepaalRolMedewerker(it, zaak),
                         explanation
                     )
-                }
+                } ?: zrcClientService.deleteRol(zaak, BetrokkeneType.MEDEWERKER, explanation)
```

Comment:
Add a unit test for this scenario? We have quite some test coverage already so should be quite easy.